modules/ptp-dual-ports-oc.adoc

```diff
@@ -21,7 +21,20 @@ In this configuration, the ports in a dual-port NIC operate as follows:
 
 * If both ports become faulty, the clock state moves to the `HOLDOVER` state, then the `FREERUN` state when the holdover timeout expires, before resyncing to a leader clock.
 
+.Hardware requirements
+
+You can configure PTP ordinary clocks with added redundancy on x86 architecture nodes with dual-port NICs.
+
+For AArch64 architecture nodes, you can use the following dual-port NICs only:
+
+* NVIDIA ConnectX-7 series
+
+* NVIDIA BlueField-3 series, in NIC mode
++
 [NOTE]
 ====
-You can configure PTP ordinary clocks with added redundancy on `x86` architecture nodes with dual-port NICs only.
+You must configure the NVIDIA BlueField-3 series DPU in NIC mode before configuring the interface as an ordinary clock with improved redundancy. For further information about configuring NIC mode, see link:https://docs.nvidia.com/networking/display/bluefielddpubspv422/modes+of+operation#src-141856548_ModesofOperation-NICModeforBlueField-3[NIC Mode for BlueField-3] (NVIDIA documentation).
```

Comment:
Should we add a note about the new required Nvidia driver for this feature? This allows mellanox cards to expose 1 phc per NIC instead of multiple (virtual) ones. Intel cards are already exposing only one phc per nic. See https://issues.redhat.com/browse/RHEL-87775